gdk4-win32/src/auto/win32_surface.rs

```diff
@@ -0,0 +1,70 @@
+// This file was generated by gir (https://github.com/gtk-rs/gir)
+// from gir-files (https://github.com/gtk-rs/gir-files.git)
+// DO NOT EDIT
+
+use glib::object::IsA;
+use glib::translate::*;
+use std::fmt;
+
+glib::wrapper! {
+    #[doc(alias = "GdkWin32Surface")]
+    pub struct Win32Surface(Object<ffi::GdkWin32Surface, ffi::GdkWin32SurfaceClass>) @extends gdk::Surface;
+
+    match fn {
+        type_ => || ffi::gdk_win32_surface_get_type(),
+    }
+}
+
+impl Win32Surface {
+    #[doc(alias = "gdk_win32_surface_get_handle")]
+    #[doc(alias = "get_handle")]
+    pub fn handle(window: &impl IsA<gdk::Surface>) -> win32::HGDIOBJ {
+        assert_initialized_main_thread!();
+        unsafe { ffi::gdk_win32_surface_get_handle(window.as_ref().to_glib_none().0) }
+    }
+
+    #[doc(alias = "gdk_win32_surface_get_impl_hwnd")]
+    #[doc(alias = "get_impl_hwnd")]
+    pub fn impl_hwnd(window: &impl IsA<gdk::Surface>) -> win32::HWND {
+        assert_initialized_main_thread!();
+        unsafe { ffi::gdk_win32_surface_get_impl_hwnd(window.as_ref().to_glib_none().0) }
+    }
+
+    #[doc(alias = "gdk_win32_surface_is_win32")]
+    pub fn is_win32(window: &impl IsA<gdk::Surface>) -> bool {
+        assert_initialized_main_thread!();
+        unsafe {
+            from_glib(ffi::gdk_win32_surface_is_win32(
+                window.as_ref().to_glib_none().0,
+            ))
+        }
+    }
```

Comment:
shouldn't those methods take a self?

Reply:
I have fixed some annotations and deprecated some functions: https://gitlab.gnome.org/GNOME/gtk/-/merge_requests/4977